src/utils.ts

```diff
@@ -15,31 +20,36 @@ export function concat(a: ByteArray, b: ByteArray): ByteArray {
   return out as ByteArray;
 }
 
-export function isZero(num: BigInt): boolean {
-  for (let i = 0; i < num.length; i++) {
-    if (num[i] != 0) {
-      return false;
-    }
-  }
-  return true;
+export function eventId(event: EthereumEvent): string {
+  return crypto
+    .keccak256(
+      concat(event.transaction.hash, event.transactionLogIndex as ByteArray),
+    )
+    .toHex();
+}
+
+export function hexToAddress(hex: string): Address {
+  return Address.fromString(hex.substr(2));
+}
+
+let debugId = 0;
+export function debug(msg: string): void {
```

Comment:
what is it and when it will be used ? 
please document that .
should it be part of the product ?

Reply:
no, since you can't use `console.log` inside a mapping, this is a workaround for that. you can use `debug(..)` and view the messages as entities in graphiql.

Reply:
please document that

Reply:
https://github.com/daostack/subgraph/pull/29/commits/06a957ac8a4fa01aa1c17139fe6d1d4c62e250cd